extensions/spark/kyuubi-spark-jvm-quake/src/main/scala/org/apache/spark/kyuubi/jvm/quake/SparkJVMQuake.scala

```diff
@@ -0,0 +1,143 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.kyuubi.jvm.quake
+
+import java.io.File
+import java.lang.management.ManagementFactory
+import java.util.concurrent.TimeUnit
+
+import scala.collection.JavaConverters._
+
+import SparkJVMQuake._
+import com.sun.management.HotSpotDiagnosticMXBean
```

Comment:
Does the plugin rely on the HotSpot JVM only classes? What happen if it's running with other JVM implementations like OpenJDK or Azul ?

Reply:
Both openJDK and Azul JDK are supported, and different versions such as JDK8, JDK11, JDK17, and JDK 21 are also supported

Reply:
Is there API documents supporting this conclusion？

Reply:
I couldn't find the corresponding API documentation, but all tests were okay